tests/kernel/fpu_sharing/float_disable/testcase.yaml

```diff
@@ -16,5 +16,5 @@ tests:
     tags: kernel userspace
   kernel.fpu_sharing.float_disable.x86:
     extra_args: CONF_FILE=prj_x86.conf
-    platform_allow: qemu_x86
+    platform_allow: qemu_x86 qemu_x86_nokpti
```

Comment:
I would just change this to `arch_allow: x86` this scenario should work on any x86 target